index.js

```diff
@@ -1,7 +1,20 @@
 const readline = require('readline-sync');
+const calc = require('./calculator');
 
 console.log('Welcome to the calculator!');
 
-console.log('Please enter some input:');
-const response = readline.prompt();
-console.log('You entered: ' + response);
+console.log('Enter the operator:');
+const operator = readline.prompt();
+
+console.log(`How many numbers do you want to ${operator} ?`);
+const c = parseInt(readline.prompt());
+
+const calculator = calc.create_calculator(operator);
+
+for (var i = 0; i < c; i++) {
+  console.log('Enter a number:');
+  const n = parseInt(readline.prompt());
```

Comment:
How come this uses parseInt? Why not parseFloat? The current code prevents inputting rationals such as 0.5, despite the fact that all the code downstream would handle this just as well as integers.

Reply:
(To add to this point: JavaScript does not distinguish between integer and float number types. As such you can use one anywhere you would the other).